core/server/master/src/main/java/alluxio/master/file/meta/InodeTreePersistentState.java

```diff
@@ -119,6 +133,31 @@ public InodeTreePersistentState(InodeStore inodeStore, InodeLockManager lockMana
     mBucketCounter = new BucketCounter(
         ServerConfiguration.getList(MASTER_METRICS_FILE_SIZE_DISTRIBUTION_BUCKETS, ",")
             .stream().map(FormatUtils::parseSpaceSize).collect(Collectors.toList()));
+    mRetryCacheEnabled =
+        ServerConfiguration.getBoolean(MASTER_FILE_SYSTEM_OPERATION_RETRY_CACHE_ENABLED);
+    mCache = CacheBuilder.newBuilder()
+        .maximumSize(ServerConfiguration.getInt(MASTER_FILE_SYSTEM_OPERATION_RETRY_CACHE_SIZE))
+        .build();
```

Comment:
just a consideration to reduce the configuration footprint, could you make the cache enabled if the size > 0 and disabled if <= 0? This way you only need one config key

Reply:
I'm favoring to keep this as is. In the future, we might need to change the eviction model to a timeout.